src/renderer/Renderer.ts

```diff
@@ -112,6 +119,46 @@ export class Renderer {
         }
     }
 
+    /**
+     * Adds a light in the `lightPositions` match an entry in the `this.lights` array.
+     *
+     * @param {REGL.Vec3} lightPositions A vector of length 3 denoting the x, y, z point where the light resides in the
+     *     vector-space.
+     * @param {REGL.Vec3} lightColors A vector of length 3 denoting _.
+     * @param {REGL.Vec3} lightIntensities A vector of length 3 denoting _.
```

Comment:
Could use some help with these descriptions @davepagurek 😳

Reply:
- `lightColor` is a `vec3` representing the r, g, and b values of the light's color, where each channel is in [0, 1]
- `lightIntensity` is a number affecting specular reflection where a lower number produces a softer, larger reflection and a higher number produces a smaller, harder reflection